modules/ROOT/pages/architecture-visualization.adoc

```diff
@@ -0,0 +1,44 @@
+= Architecture Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== About the Architecture Visualization
+
+The Architecture Visualization shows the topology of your application network. In this visualization, you can assign change display names of nodes, add tags, and assign nodes to layers. The Architecture Visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.
+
+== View Information about One Service in your Network
+
+When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are unavailable.
+Anypoint Visualizer displays information about the node in a card. This card is divided into the following tabs:
+
+* *Details*: Provides information about the node.
+* *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can xref:customize-your-canvas.adoc#display-name[edit the node's display name] or xref:layers.adoc#set-a-layer-ui[change the layer] and xref:use-tags-in-visualizer.adoc[tags] from this section.
+
+== Filtering your View
+
+When you select a node, you can click *Select dependents and dependencies*, which selects all nodes related to the service you selected.
```

Comment:
```suggestion
When you select a node (service) that has dependents and dependencies, you have the option to also select all nodes related to the service you selected. If the node you select does not have dependents and dependencies, this option is not offered.
```
Do we have to equate node with service? If possible, suggest using just one term throughout.
I moved this parenthetical up from where it appears below.